libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/ActionButtonBinder.kt

```diff
@@ -56,4 +59,35 @@ internal class ActionButtonBinder(
             }
         )
     }
+
+    private fun audioGuidanceButtonComponent(
+        binding: MapboxActionButtonsLayoutBinding,
+        style: Int,
+        store: Store
+    ): AudioGuidanceButtonComponent {
+        return AudioGuidanceButtonComponent(binding.soundButton, style, contractProvider = {
+            DropInAudioComponentContract(context.viewModel.viewModelScope, store)
+        })
+    }
+}
+
+@ExperimentalPreviewMapboxNavigationAPI
+internal class DropInAudioComponentContract(
```

Comment:
We are not using the term `DropIn` in our codebase. I don't have any better name suggestions. Let's just change it to `AudioComponentContractImpl`
@tomaszrybakiewicz